src/engraving/tests/playbackmodel_tests.cpp

```diff
@@ -373,6 +373,87 @@ TEST_F(Engraving_PlaybackModelTests, Spanners)
     }
 }
 
+/**
+ * @brief PlaybackModelTests_Dynamics
+ * @details Test simple dynamic markings and hairpins
+ */
+
+TEST_F(Engraving_PlaybackModelTests, Dynamics)
```

Comment:
When https://github.com/musescore/MuseScore/pull/19289 is merged, this unit test should test PlaybackContext, not PlaybackModel